tests/testthat/test_are_all_close.R

```diff
@@ -0,0 +1,39 @@
+testthat::test_that(
+  "Return True when two vectors are close", {
+    set.seed(100)
+    x = runif(100, min = 1, max = 10)
```

Comment:
-1

If possible, don't use complex test cases. Use something simple that is as easily verifiable as possible. If you have to debug this, it's much harder to figure out what's going if `x=runif(100, min = 1, max = 10)` compared to `x=c(1,2)`

If you do want to test a bunch of cases and want to generate random sizes that's also ok, but I'd add them as a additional test cases, and always remember to set the seed as you've done.